templates/docker-compose/.env

```diff
@@ -5,19 +5,28 @@ HTTP_REMOTE_PORT=80
 HTTPS_REMOTE_PORT=443
 MYSQL_REMOTE_PORT=3306
 REDIS_PORT=6379
+MEILISEARCH_PORT=7700
 FRONTEND_NETWORK=supportpal_frontend
 BACKEND_NETWORK=supportpal_backend
 
 # Database
 MYSQL_DATABASE=supportpal
 
+# Meilisearch
+MEILISEARCH_HOST=http://${MEILISEARCH_SERVICE_NAME}:${MEILISEARCH_PORT}
+MEILISEARCH_KEY=qeh2pZJIKTJGBHFkFIq7XKpUkTmRj1
+MEILI_ENV=production
+MEILI_LOG_LEVEL=WARN
+MEILI_SCHEDULE_SNAPSHOT=86400
```

Comment:
These need moving to the end of the file, or at least after `MEILISEARCH_SERVICE_NAME` is defined